File: TealiumInitializer.cs
using UnityEngine;
using System.Collections;
using System.Collections.Generic;

public class TealiumInitializer : MonoBehaviour {
		
	void Start() {
		Tealium.TrackView("First Level", new Dictionary<string, string>() {
			{"custom_alpha", "alpha"}
			, {"custom_beta", "beta"}
		});
	}
}

File: TealiumInitializer.js
#pragma strict

function Awake() {
	Tealium.Initialize("tealiummobile", "demo", "dev", 
		new Tealium.Config()
			.SilenceLogs(false));
}

function Start() {
	var data = new Dictionary.<String, String>(2);
	data["custom_alpha"] = "alpha";
	data["custom_beta"] = "beta";
	
	Tealium.TrackView("First Level", data);
	
}